src/commands/dev/dev.js

```diff
@@ -417,7 +419,8 @@ const validateGeoCountryCode = (arg) => {
  */
 const dev = async (options, command) => {
   log(`${NETLIFYDEV}`)
-  const { api, config, repositoryRoot, site, siteInfo, state } = command.netlify
+  const { api, cachedConfig, config, repositoryRoot, site, siteInfo, state } = command.netlify
+  const netlifyBuild = await netlifyBuildPromise
```

Comment:
NIT: maybe add todo here:

```suggestion
  // TODO: once ESModule Migration is done we can remove that`
  const netlifyBuild = await netlifyBuildPromise
```

Reply:
I can add it if you feel strongly about it, but we already have this pattern elsewhere in the application without accompanying comments (e.g. https://github.com/netlify/cli/blob/main/src/lib/build.js#L47). I think these instances will be pretty easy to find once we do the migration.